src/pages/Favorites/Favorites.jsx

```diff
@@ -0,0 +1,46 @@
+import FavoriteMovie from "./FavoriteMovie";
+import { useEffect, useState } from "react";
+import { FAVORITE_MOVIES } from "../../constants/constants.js";
+
+import styles from "./Favorites.module.scss";
+
+export const Favorites = () => {
+  const [favoriteMovies, setFavoriteMovies] = useState([]);
+
+  const getFavoriteMovies = async () => {
```

Comment:
u dont need async right here cause this function doesnt have an async actions inside. Actually, u dont even need this function, just move the 11-12 lines code to the useEffect directly